config/application.php

```diff
@@ -120,6 +130,6 @@
 /**
  * Bootstrap WordPress
  */
-if (!defined('ABSPATH')) {
-    define('ABSPATH', $webroot_dir . '/wp/');
+if (! defined('ABSPATH')) {
```

Comment:
Lol I hate this space after prefixed unary operators. Is this our coding style now?

Reply:
https://www.php-fig.org/psr/psr-1/#23-side-effects scroll down little bit to conditional example. /shrug – seems to be the norm now